src/main/java/com/growup/todoary/exception/GlobalExceptionHandler.java

```diff
@@ -8,8 +8,8 @@
 @RestControllerAdvice
 public class GlobalExceptionHandler {
 
-    @ExceptionHandler(NotExistentTodoException.class)
-    public ResponseEntity<String> notExistentTodo(NotExistentTodoException e) {
+    @ExceptionHandler(TodoNotFoundException.class)
+    public ResponseEntity<String> notExistentTodo(TodoNotFoundException e) {
```

Comment:
함수명을 다음과 같이 변경하는건 어떨까요?

```suggestion
    public ResponseEntity<String> todoNotFoundHandler(TodoNotFoundException e) {
```

Reply:
수정하겠습니다!